examples/persistent_homology_graphs.ipynb

```diff
@@ -0,0 +1,659 @@
+{
```

Comment:
<p>Small suggestions:</p><ul><li>"giotto-tda can extract topological features from ..."</li><li>Remove the parentheses () from "...directed graphs and with <code>directed=False</code> ..."</li><li>Remove "Another important point is that" and just simplify the sentence as follows "Although adjacency matrices of sparsely connected or unweighted graphs can be passed directly to these transformers, they are interpreted as <em>weighted</em> adjacency matrices according to some non-standard conventions. We will highlight these conventions below."</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/giotto-ai/giotto-tda/pull/462/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
<p>Done.</p>